source/sonata_tech.rst

```diff
@@ -584,16 +584,16 @@ The format of the weights file is described below:
-  /electrodes/{array_name}/{electrodename}/{population_name}    index                   uint64                         Mandatory     Column corresponding to this electrode in /electrodes/{population_name}/scaling_factors
-  /electrodes/{population_name}                                 scaling_factors         float64    Total_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
-  /{population_name}                                            node_ids                uint64     N_nodes             Mandatory     List of node ids
-  /{population_name}                                            section_ids             uint64     Total_seg           Mandatory     For each segment, index of the section to which it belongs. Ordered by section ids and grouped by nodes.
-  /{population_name}                                            offsets                 uint64     N_nodes + 1         Mandatory     The offset for each node in the data and sec_ids fields
-  ===========================================================   ======================= ========== =================== ============= ==========================================
+  ============================================= ======================= ========== =================== ============= ==========================================
+  Group                                         Field                   Type       Shape               Requirement   Description
+  ============================================= ======================= ========== =================== ============= ==========================================
+  /electrodes/{electrodename}                   layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/{electrodename}                   position                float32    3                   Mandatory     Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/{electrodename}                   region                  utf8       1                   Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}                   type                    utf8       1                   Optional      Either EEG or LFP
+  /electrodes/{electrodename}/{population_name} index                   uint64     1                   Mandatory     Index of the column corresponding to this electrode in /electrodes/{population_name}/scaling_factors
+  /electrodes/{population_name}                 scaling_factors         float64    Total_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
+  /{population_name}                            node_ids                uint64     N_nodes             Mandatory     List of node ids
```

Comment:
does this have to be all the node ids in the population?

Reply:
No, the ones not here will be ignored during the simulation. Basically the report will have the intersection.

Reply:
Ah, I see. What happens if the different electrodes require different node ids?

Reply:
@jorblancoa / @joseph-tharayil ?

Reply:
We don't support different electrodes to report on different node ids

Reply:
Is there a physical/scientific reason for that?

This should probably describe then; something to the effect of `Electrode files are organized to have all the same NodeIds`

I would note that in the nodes and edges files, different populations are organized within the same file, and the same could be done here so that groups of electrodes with different node IDs can be supported by shifting things right; something like:
```
/electrodes/{electrode_group}/{electrodename}/[same as before]
/electrodes/{electrode_group}/{population_name}/node_ids
```